index.html

```diff
@@ -32,7 +32,145 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
+ </ul>  
+ <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="project-1 grid-project">
+  <div class="multi-post">
+   <h3 class="multi-text-color">Multi-Post Stories</h3>
+   <h3 class="multi-text-color">Gain+Glory</h3>
+   <ul class="language display-flex">
+   <li class="language-background">Ruby on rails</li>
+   <li class="language-background">css</li>
+   <li class="language-background" >JavaScript</li>
+   <li class="language-background">html</li>    
+ </ul>  
+ <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="project-1 grid-project">
+   <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories</h3>
+    <h3 class="multi-text-color">Gain+Glory</h3>
+    <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>  
+  <button class="see-project">See Project</button>
+</div>
+</div>
+</section>
+
+<!-- About me Session -->
+<section class="about-container">
+    <img class="float-right" src="images/Mask_Group_floate_right.png" alt="elipse-floating-right">  
+      <h2 class="about-me">About Me</h2><br><br><br>
+         <p class="about-message">I can help you build a product, feature or website 
+         Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</p>
+         <div><img class="float-left" src="images/Group_float_left.png" alt="floating-elispe-left">
+  <button class="see-project add-bottom-margin">Get My Resume</button>  
+</div> 
+</section>
+
+<!-- Language Session -->
+<section class="grid-frame-skill">
+<div class="top-padding grid-child-frame-skill">  
+    <img class="skills-pack" src="images/globe.png" alt="globe-logo"><br><br><br>
+    <h2 class="skills-pack-font">Languages</h2><br>
+    <ul class="skill-flex display-flex">
+    <li class="frame-skill-background">JavaScript</li>
+    <li class="frame-skill-background">Ruby</li>
+    <li class="frame-skill-background">CSS</li>
+    <li class="frame-skill-background">Html</li>    
+  </ul>
+</div>
+<!-- Frameworks Session -->
+<div class="top-padding grid-child-frame-skill">  
+   <img class="skills-pack" src="images/framework.png" alt="Frameworks"><br><br><br>
+    <h2 class="skills-pack-font">Frameworks</h2><br>  
+    <ul class="skill-flex display-flex">
+    <li class="frame-skill-background">Bootstrap</li>
+    <li class="frame-skill-background">Ruby on rails</li>
+    <li class="frame-skill-background">Rspec</li>
+    <li class="frame-skill-background">Capybara</li>
+    <li class="frame-skill-background">Selenium</li>     
+  </ul>  
+</div>
+<!-- Skill Session -->
+<div class="top-padding grid-child-frame-skill">  
+    <img class="skills-pack" src="images/skills.png" alt="Skill-logo"><br><br><br>
```

Comment:
### These requirements from the previous CR haven't been addressed yet, I'll give some tips with each one of them on how to fix it:
- [ ] Your tech stacks are somewhat aligned to the right, whereas on the template they are centered. Kindly fix that.
  - in order to fix this you should remove the `padding-left` which is part of the automatic styling for unordered lists for example by doing 
`.skill-flex {padding-left: 0;}`

- [ ] Kindly ensure you use the appropriate background color as we have on the template.
  - I think this one was just forgot as it speaks for itself.

- [ ] Kindly ensure you implement the appropriate spaces around the edges and the bottom of the containers.
  - in order to fix this I suggest you use Margin instead of padding to fix it.

Reply:
Hey there, you are a genius; I love the fact that you suggested possible solutions. Thank a lot